manifest.appcache

```diff
@@ -1,3 +1,6 @@
+# Random number: 23
```

Comment:
How about a number that steadily increments instead of a random number?

It could also serve as a sort of a version marker.

It could start with 1, and then increment every time we want to update.